packages/esm-patient-chart-app/src/visit-header/visit-header.component.tsx

```diff
@@ -206,23 +208,27 @@ const VisitHeader: React.FC = () => {
             <PatientInfo patient={patient} />
           </div>
           <HeaderGlobalBar>
-            <ExtensionSlot extensionSlotName="visit-header-right-slot" />
-            {!hasActiveVisit && !isDeceased && (
-              <Button className={styles.startVisitButton} onClick={launchStartVisitForm} size="lg">
-                {startVisitLabel ? startVisitLabel : t('startAVisit', 'Start a visit')}
-              </Button>
-            )}
-            {currentVisit !== null && endVisitLabel && (
+            {systemVisitEnabled && (
               <>
-                <HeaderGlobalAction
-                  className={styles.headerGlobalBarButton}
-                  aria-label={endVisitLabel ?? t('endAVisit', 'End a visit')}
-                  onClick={() => openModal(patient?.id)}
-                >
-                  <Button as="div" className={styles.startVisitButton}>
-                    {endVisitLabel ? endVisitLabel : <>{t('endAVisit', 'End a visit')}</>}
+                <ExtensionSlot extensionSlotName="visit-header-right-slot" />
+                {!hasActiveVisit && (
```

Comment:
Removing `!isDeceased` will make start visit button to be displayed on the deceased patient.